api/src/db/migrations/v82_add_up_and_running_owning_operating_phase.ts

```diff
@@ -0,0 +1,318 @@
+import { v81UserData } from "./v81_add_completed_filing_payment";
+
+export interface v82UserData {
+  user: v82BusinessUser;
+  profileData: v82ProfileData;
+  formProgress: v82FormProgress;
+  taskProgress: Record<string, v82TaskProgress>;
+  taskItemChecklist: Record<string, boolean>;
+  licenseData: v82LicenseData | undefined;
+  preferences: v82Preferences;
+  taxFilingData: v82TaxFilingData;
+  formationData: v82FormationData;
+  version: number;
+}
+
+export const migrate_v81_to_v82 = (v81Data: v81UserData): v82UserData => {
+  const operatingPhase =
+    v81Data.profileData.businessPersona === "OWNING" &&
+    v81Data.profileData.operatingPhase === "UP_AND_RUNNING"
+      ? "UP_AND_RUNNING_OWNING"
+      : v81Data.profileData.operatingPhase;
+
+  return {
+    ...v81Data,
+    preferences: {
+      ...v81Data.preferences,
+      isHideableRoadmapVisible: false,
```

Comment:
I know it feels annoying but I think we should have a separate migration for adding this. Or at least change the file name to include it as well maybe?

Reply:
I'll create another migration file